### DISCUSSION TITLE
How do I hide line numbers?

### DISCUSSION THREAD

**Discussion by tm99hjkl** [helix-editor/helix#9454] [Q&A]
Hi community,

Is there a command in Helix similar to [`:set nu!` in vi](https://vimdoc.sourceforge.net/htmldoc/options.html#'nu')? 
I'd like to hide line numbers sometimes when reading source code on my laptop.

I would appreciate it if you could let me know.
Thanks for confirming :)

**Comment by woojiq** [comment]
https://docs.helix-editor.com/master/configuration.html#editorgutters-section
There is Discussion tab for HowTo questions.

**Comment by jeffs** [comment]
In config.toml:

```toml
editor.gutters = ["diff", "diagnostics", "spacer"]
```

See also:
* https://github.com/helix-editor/helix/discussions/6995
* [[editor.gutters] Section](https://docs.helix-editor.com/editor.html?highlight=gutter#editorgutters-section)
